lib/lua/os.ml

```diff
@@ -0,0 +1,30 @@
+type time
+
+external val clock : unit -> float = "os.clock"
+
+external val strftime : string -> time -> string =
+  "function(spec, time) \
+     if (spec == '*t') or (spec == '!*t') then \
+       return os.date((spec == '!*t') and '!' or '%c', time) \
+     else \
+       return os.date(spec, time) \
+     end\
+   end"
+
+external val exit_success : unit -> 'a =
+  "function() return os.exit(0) end"
+
+external val exit_failure : unit -> 'a =
+  "function() return os.exit(1) end"
+
+external val getenv : string -> string =
```

Comment:
Hrrm, would it be worth this returning an `option string` instead? I realise this'd require some additional work, as we'd need to have a way of checking and coercing values.

If not, maybe just `os.getenv(x) or ""`? Should mean we can inline it too.

Reply:
... Aye, I forgot about `or`. I don't return an `option string` here because the `lua/` modules are all "freestanding" - they don't depend on `prelude.ml`. The option is unboxing the `option` return value and taking two continuations